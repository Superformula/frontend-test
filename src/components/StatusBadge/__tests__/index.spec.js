import React from 'react';
import { shallow, mount } from 'enzyme';
import { StatusBadge } from '../';

describe('StatusBadge', () => {
  it('should render without failing', () => {
    const wrapper = mount(<StatusBadge />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render with the status of open', () => {
    const wrapper = shallow(<StatusBadge isOpen />);
    expect(wrapper).toMatchSnapshot();
  });
});
