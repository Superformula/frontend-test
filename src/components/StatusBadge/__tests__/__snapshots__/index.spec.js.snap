// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBadge should render with the status of open 1`] = `
<Badge
  isOpen={true}
>
  open
</Badge>
`;

exports[`StatusBadge should render without failing 1`] = `
.c0 {
  display: inline-block;
  border-radius: 40px;
  padding: 0 6px;
  text-transform: uppercase;
  -webkit-letter-spacing: 0.57px;
  -moz-letter-spacing: 0.57px;
  -ms-letter-spacing: 0.57px;
  letter-spacing: 0.57px;
  line-height: 16px;
  font-size: 8px;
  font-weight: 500;
  background-color: #FF3548;
  color: #FFF;
}

<StatusBadge>
  <Badge>
    <span
      className="c0"
    >
      closed
    </span>
  </Badge>
</StatusBadge>
`;
